lib/wrapper.rb

```diff
@@ -0,0 +1,29 @@
+module Wrapper 
```

Comment:
Would you mind calling it LogWrapper, so it's more clear when you call it?

Reply:
Yes.